Example/ViewController.swift

```diff
@@ -154,6 +154,8 @@ class ViewController: UIViewController {
         navigationMapView.removeRoutes()
         navigationMapView.removeWaypoints()
         waypoints.removeAll()
+        navigationMapView.mapView.locationManager.locationProvider.startUpdatingHeading()
+        navigationMapView.mapView.locationManager.locationProvider.startUpdatingLocation()
```

Comment:
Similarly, pressing the Clear button shouldn’t affect how the map tracks the user’s location – that button is purely about the route-related overlays that appear on the map.